package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "tiddlywiki",
   "preferGlobal": "true",
-  "version": "5.1.16-prerelease",
+  "version": "5.1.16",
```

Comment:
5.1.16 is already released. So it needs to be 5.1.17-prerelease now.

Reply:
The issue is 5.1.16 was released without removing the prerelease flag.  So stable 5.1.16 lists it's version as 5.1.16-prerelease.  This is an attempt to fix that.

Reply:
See https://github.com/Jermolene/TiddlyWiki5/issues/3239

Reply:
ahh .. OK. .. my bad